fpgaconvnet/parser/onnx/passes.py

```diff
@@ -236,8 +236,10 @@ def remove_training_nodes(model):
                 as it is not used for inference")
 
         # get the next node
-        next_node = next(filter(lambda x: x.input[0] == node.output[0], model.graph.node))
-
+        try:
```

Comment:
I know this cannot be the best way to fix it, but might be the fastest

Reply:
I agree. Lets move on using this fix for the moment and we can refine this one at a later point if needed